setup.py

```diff
@@ -107,7 +107,8 @@
   install_requires = ['six>=1.11.0', 'subprocess32; python_version < "3"'],
   extras_require = {
       'colors': ['colorama>=0.3.9'],
-      'crypto': ['cryptography>=2.6'],
+      'crypto:python_version < "3"': ['cryptography>=2.6,<3.4'],
+      'crypto:python_version >= "3"': ['cryptography>=2.6'],
```

Comment:
```suggestion
      'crypto:python_version < "3"': ['cryptography>=3.3.2,<3.4'],
      'crypto:python_version >= "3"': ['cryptography>=3.3.2'],
```
... for CVE-2020-36242?